rotkehlchen/constants/limits.py

```diff
@@ -3,3 +3,6 @@
 FREE_HISTORY_EVENTS_LIMIT: Final = 100
 FREE_ETH_TX_LIMIT: Final = 100
 FREE_USER_NOTES_LIMIT: Final = 10
+
+# Used for tests or where we don't need to enforce the premium tier limits
+INTERNAL_QUERY_LIMIT: Final = 9_223_372_036_854_775_807  # 2^63 - 1
```

Comment:
Yes this is bad. This is a production constant used for tests. This needs to change 100%.